inji-verify/src/components/Home/VerificationSection/ScanQrCode.tsx

```diff
@@ -45,19 +46,20 @@ const Upload = () => (
 
 const ScanQrCode = () => {
     const method = useVerificationFlowSelector(state => state.method);
+    const outline = window._env_.DEFAULT_THEME !=="purple_theme" ? orangeScanOutline : purpleScanOutline;
```

Comment:
can we move this also to a common config that provides proper asset based on theme?

Reply:
Moved to config file